doc/server-reflection.md

```diff
@@ -15,6 +15,19 @@ This broadly involves two problems: determining what formats (which protobuf
 messages) a server’s method uses, and determining how to convert messages
 between human readable format and the (likely binary) wire format.
 
+## Enabling server reflection
```

Comment:
I think adding links to tutorials in different languages is very useful, but at the same time I'm not 100% convinced that this document is the right place for it (or at least the way it's being done).
This doc is the "gRPC server reflection protocol" and it was meant mostly as a design doc / spec (and not a guide how to use), so suddenly adding a section that links to tutorials feels a bit misplaced).

Btw, there's also the C++ server tutorial which ideally could act as general introduction but, at the same time it's C++ specific, despite lot of the concepts described there (and the grpc_cli command examples) are universal.
Maybe it could be reworked to contain the general introduction and then link to language-specific guides?
This "protocol description" doc could then contain only a link to usage guide.
What do you think?

Reply:
Good points. I choose this doc because it has the most SEO when users search for "grpc server reflection", but I'm open to other docs. The doc you suggested - is that https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-HTTP2.md?

Alternatively, perhaps an option is a spot on grpc.io.

Reply:
Perhaps this can be moved to the bottom of the page under some sort of "Known Implementations" section?   I kind of agree with Jan here, since I believe this doc explains what reflection should do.  (i.e. this feels like a design doc)

Reply:
Bottom of the page would make sense for me, too. I'll do so tomorrow unless there are other thoughts on the issue by then.

Reply:
I think moving to the bottom works. Thanks!